packages/filecoin-rpc-client/src/index.ts

```diff
@@ -68,12 +54,9 @@ export default class LotusRpcEngine {
         jsonrpc: '2.0',
         method: `Filecoin.${method}`,
```

Comment:
I think we should pass an optional `namespace` as a `LotusRPCEngineConfig` param - the default value should be `Filecoin`, but in the future, we can use other namespaces because (as far as I know) this is how the FVM team plans on making other RPC's (like the FEVM RPC) available over lotus

The `method` would then become:

```
method: `${config.namespace}.${method}`
```

Reply:
Sounds good, shall I add it to this PR?

Reply:
Added it! Was just a small effort :)